portfolio/src/main/java/com/google/sps/utils/CommentDataStore.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.utils;
+
+import com.google.appengine.api.datastore.*;
+import com.google.sps.data.Comment;
+import com.google.sps.data.EntityConvertible;
+import com.google.sps.data.EntityConvertibleCreator;
+
+/** Provides the service to interact with Datastore to store or load comment. */
+public class CommentDataStore {
+
+    static class Constants {
+        final static String KIND = "Comment";
+        final static String TIMESTAMP = "timestamp";
+    }
+
+    /**
+     * Puts the entity into Datastore.
+     * @param comment Comment object needed to be stored.
+     */
+    public static void store(Comment comment) {
+        DatastoreService datastoreService = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Let's not have this class create its own underlying data store. Get it injected from constructor instead.

public CommentDataStore(DatatoreService datastoreService) {
   this.datastoreService = datastoreService;
}

This way this class depends on the interface, not the implementation of the data store.

Reply:
Fixed, but why "This way this class depends on the interface, not the implementation of the data store." ?

Reply:
Since I make changes on the code structure, I simply make it as a private variable

Reply:
As discussed offline, do not depend on how datastore is created (maybe not an instance from dataservicefactory but just a fake datastore for testing)